lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -201,6 +201,52 @@ describe('ReportRenderer', () => {
       assert.ok(/\dx/.test(descriptionsTxt), 'should have added CPU');
       assert.ok(descriptionsTxt.includes(sampleResults.userAgent), 'user agent populated');
     });
+
+    describe('axe-core', () => {
+      let axe;
+
+      beforeAll(() =>{
+        // Needed by axe-core
+        // https://github.com/dequelabs/axe-core/blob/581c441c/doc/examples/jsdom/test/a11y.js#L24
+        global.window = global.self;
+        global.Node = global.self.Node;
+        global.Element = global.self.Element;
+
+        axe = require('axe-core');
+      });
+
+      afterAll(() => {
+        global.window = undefined;
+        global.Node = undefined;
+        global.Element = undefined;
+      });
+
+      it('renders without axe violations', (done) => {
+        const container = renderer._dom._document.body;
+        const output = renderer.renderReport(sampleResults, container);
+
+        const config = {
+          rules: {
+            // Reports may have duplicate ids
+            // https://github.com/GoogleChrome/lighthouse/pull/5058#discussion_r184791923
```

Comment:
Should we convert that comment into a bug, and the reference it here as a TODO? @brendankenny

Reply:
Yeah, that's going to take a bit of work to fix (and thinking on how we'd want to dedupe audit IDs), though I think the main thing depending on that behavior is tests, so hopefully it won't be too gross to fix. Makes sense as an issue.

Way to find that PR comment, @johnemau :)

Reply:
I created the issue: https://github.com/GoogleChrome/lighthouse/issues/9432